framework/src/org/checkerframework/common/reflection/reflection.astub

```diff
@@ -0,0 +1,29 @@
+import org.checkerframework.common.reflection.qual.*;
+
+package java.lang;
+
+class Class {
+    @ForName static Class<?> forName(String name);
```

Comment:
Method declaration annotations should be on a separate line above the method:
@ForName
static Class<?>...